WooCommerce/Classes/ViewModels/Order Details/OrderDetailsViewModel.swift

```diff
@@ -265,12 +269,18 @@ extension OrderDetailsViewModel {
     }
 
     func syncTrackingsEnablingAddButtonIfReachable(onReloadSections: (() -> ())? = nil, onCompletion: (() -> Void)? = nil) {
-        syncTracking { [weak self] error in
-            if error == nil {
-                self?.trackingIsReachable = true
+        // If the plugin is not active, there is no point on continuing with a request that will fail.
+        isPluginActive(SitePlugin.SupportedPlugin.WCTracking) { [weak self] isActive in
+            guard let self = self, isActive else {
+                onCompletion?()
+                return
+            }
+
+            self.trackingIsReachable = true
+            self.syncTracking { error in
```

Comment:
`error` is never checked and `trackingIsReachable` is always true?

Reply:
`trackingIsReachable` starts with a `false` value and we set it to `true` here because at this point we already validated that the plugin is active.

If there is an error, the error happens when syncing tracking providers. Does that make sense?

Reply:
So you're deciding earlier about availability here? Got it, thanks for explanation.